recipes/heavydb/build-heavydb-common.bat

```diff
@@ -0,0 +1,56 @@
+@echo on
+
+@REM Contains a patch for arrow-cpp
+@REM https://github.com/apache/arrow/pull/11806
+COPY %RECIPE_DIR%\arrow_aligned_storage.h %PREFIX%\Library\include\arrow\util\aligned_storage.h
```

Comment:
@guilhermebs Do we still need it? Or, which arrow-cpp version contains the fix?

Reply:
I think you tagged the wrong person :)

Reply:
Indeed sorry about that.

@guilhermeleobas can you have a look?

Reply:
@pearu I think this issue was solved in arrow 7.*

See: https://issues.apache.org/jira/browse/ARROW-14800